### PR TITLE
[FIX] Fix mistake in FASTConvLayer and tf reparameterization

### PR DESCRIPTION
**Pull request by felixdittrich92**
This PR:

- Fix mistake in FASTConvLayer (special thanks to @czczup (FAST author) for finding this mistake)
- Fix tensorflow reparameterization
- Provide new pretrained backbone checkpoints
- Add small test to check that reparam works as expected

Any feedback is welcome :hugs: 

@odulcy-mindee Please upload the checkpoints before :)

### CODE REVIEW

**Comment by felixdittrich92** [comment]
@odulcy-mindee We should check the `shrink_ratio` with the latest checkpoint before :)

**Comment by czczup** [comment]
This is really great. Thanks for your time and effort!

File: doctr/models/classification/textnet/pytorch.py [hunk 1]
```diff
@@ -22,21 +22,21 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (3, 32, 32),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_tiny-c5970fe0.pt&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_tiny-27288d12.pt",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/pytorch.py:25]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_tiny-27288d12.pt&src=0",
```

File: doctr/models/classification/textnet/pytorch.py [hunk 2]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_small": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (3, 32, 32),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_small-6e8ab0ce.pt&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_small-43166ee6.pt",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/pytorch.py:32]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_small-43166ee6.pt&src=0",
```

File: doctr/models/classification/textnet/pytorch.py [hunk 3]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_base": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (3, 32, 32),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_base-8295dc85.pt&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_base-7f68d7e0.pt",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/pytorch.py:39]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_base-7f68d7e0.pt&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 4]
```diff
@@ -22,21 +22,21 @@
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_tiny-9e605bd8.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_tiny-fe9cc245.zip",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:25]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_tiny-fe9cc245.zip&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 5]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_small": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_small-4784b292.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_small-29c39c82.zip",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:32]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_small-29c39c82.zip&src=0",
```

File: doctr/models/classification/textnet/tensorflow.py [hunk 6]
```diff
@@ -22,21 +22,21 @@
...
     },
     "textnet_base": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (32, 32, 3),
         "classes": list(VOCABS["french"]),
-        "url": "https://doctr-static.mindee.com/models?id=v0.7.0/textnet_base-2c3f3265.zip&src=0",
+        "url": "https://github.com/mindee/doctr/releases/download/v0.8.1/textnet_base-168aa82c.zip",
```
Comment (odulcy-mindee) [inline doctr/models/classification/textnet/tensorflow.py:39]:
```suggestion
        "url": "https://doctr-static.mindee.com/models?id=v0.8.1/textnet_base-168aa82c.zip&src=0",
```

**Review by odulcy-mindee — COMMENTED** [review]
The uploaded checkpoints ;-)

**Review by odulcy-mindee — APPROVED** [review]
Thanks !

**Pull request merged**